src/screens/helping/services.jsx

```diff
@@ -0,0 +1,162 @@
+      <HeaderContainer>
+        Service Information
+      </HeaderContainer>
+      
+      <FormSection>
+        <StackInput required label="Your Name:">
+          <Input
+            onChange={({ target }) => update('name', target.value)}
+          />
+        </StackInput>
+
+        <StackInput required label="Organization/Company:">
+          <Input
+            onChange={({ target }) => update('update-this-field', target.value)}
+          />
+        </StackInput>
+
+        <StackInput required label="Phone Number:">
+          <Input
+            onChange={({ target }) => update('phoneNumber', target.value)}
+          />
+        </StackInput>
+
+        <StackInput required label="Email Address:">
+          <Input
+            onChange={({ target }) => update('emailAddress', target.value)}
+          />
+        </StackInput>
+
+        <StackInput label="Website:">
+          <Input
+            onChange={({ target }) => update('update-this-field', target.value)}
+          />
+        </StackInput>
+      </FormSection>
+
+      <FormSection>
+        <StackInput required label="Service Fees:">
+          <RadioGroup value={formData.duration} onChange={({ target }) => update('duration', target.value)}>
+            <Radio value={'short'}>Full Fee</Radio>
+            <Radio value={'long'}>Reduced Fee</Radio>
+            <Radio value={'permanent'}>Pro Bono</Radio>
+          </RadioGroup>
+        </StackInput>
+      </FormSection>
+
+      <FormSection>
+        <StackInput required label="Description of services:">
+          <TextArea
+            placeholder="Legal counsel, Insurance, Accounting, Therapy & emotional support, House cleaning, Childcare, Transportation, Chiropractic..."
+            autosize={{ minRows: 3 }}
+            onChange={({ target }) => update('childNotes', target.value)}
+          />
+        </StackInput>
+      </FormSection>
+      
+      <FormSection>
+        <StackInput required label="Preferred Method(s) of customer contact:">
+          <CheckboxGroup 
+            options = {contactOptions}
+          />
+        </StackInput>
+      </FormSection>
+
+      <FormSection>
+        <StackInput label="Additional Information:">
+          <TextArea
+            placeholder="Additional Information..."
+            autosize={{ minRows: 2 }}
+            onChange={({ target }) => update('childNotes', target.value)}
+          />
+        </StackInput>
+      </FormSection>
+
+      <div style={{ padding: '30px 0' }}>
+        <Button
+          type="primary"
+          style={{ width: '100%', height: '44px' }}
+          onClick={() => actions.create(formData, { onSuccess: () => history.push('/looking_for_resources/housing')})}
+        >
+          Submit!
+        </Button>
+      </div>
+    </Container>
+  </Layout>
+)
+
+Services.defaultProps = {
+
+}
+
+export default compose(
+  withStateHandlers(
+    {
+      formData: {
+        housingType: 'room',
```

Comment:
We're going to want to change the default values here to match the form you've created above.

Reply:
Dummy data removed